Consortium.md

```diff
@@ -0,0 +1,51 @@
+# CreateJS Consortium
```

Comment:
This isn't in the proper format - content needs to be moved into the file below.